src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', function() {
+  const logo = document.querySelector('.logo');
+  const body = document.querySelector('body');
+
+  // Promise 1
+  const promise1 = new Promise(function(resolve, reject) {
+    logo.addEventListener('click', function() {
+      resolve();
+    });
+  });
+
+  // Promise 2
```

Comment:
not needed to add comments to your code, good code = code that doesn't need comments :)